arelle/webserver/bottle.py

```diff
@@ -263,7 +263,6 @@ def __get__(self, obj, cls):
 
 class BottleException(Exception):
     """ A base class for exceptions used by bottle. """
-    pass
```

Comment:
These seem useful to prevent accidental code from being injected. I.e.:

```
>>> def method():
...     pass
...     return 1
...
>>> return method()
  File "<stdin>", line 1
SyntaxError: 'return' outside function
```

also, bottle is actually a library that he has just dropped in here. I suggest deleting the bottle stuff and just installing the library proper so we can dependabot and what not